app/models/import.rb

```diff
@@ -0,0 +1,2 @@
+class Import < ApplicationRecord
+end
```

Comment:
What's this import about? Would it have been better to call it `RedmineImport` or `RedmineTimeEntriesImport`?